### PR TITLE
Add Meilisync PostgreSQL guide

### PR DESCRIPTION
**Pull request by Strift**
Partially addresses https://github.com/meilisearch/documentation/issues/2542

I adapted the MySQL guide to PostgreSQL by following the [guide](https://www.notion.so/meilisearch/Primary-DBs-with-Meilisearch-cf98503b6e6a40308f4e273ee828820e#106788f49fc740b7a9056ee49a61a3f4) written by @alallema.

🔗  [Preview](https://website-git-deploy-preview-mei-16-meili.vercel.app/docs/branch%3Ameilisync-postgresql/learn/cookbooks/meilisync_postgresql)

### CODE REVIEW

File: learn/primary_db_sync/meilisync_postgresql.mdx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+meilisearch:
+  api_url: MEILISEARCH_API_URL
+  api_key: 'MEILISEARCH_API_KEY'
+  insert_size: 1000
+  insert_interval: 10
+```
+</Tabs.Content>
+</Tabs.Container>
+
+Replace `MEILISEARCH_API_KEY` with an API key able to create, update, and delete documents and indexes. Replace `MEILISEARCH_API_URL` with the your instance's API URL. If you are using [Meilisearch Cloud](https://cloud.meilisearch.com), this URL should look similar to this: https://ms-4d85L33tC0d3-5041.fra.meilisearch.io
```
Comment (CaroFG) [inline learn/primary_db_sync/meilisync_postgresql.mdx:94]:
```
Replace `MEILISEARCH_API_URL` with the your instance's API URL.
```

'the' or 'your', not both

File: learn/primary_db_sync/meilisync_postgresql.mdx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+source:
+  type: postgres
+  host: 127.0.0.1
+  port: 5432
+  database: POSTGRESQL_DATABASE_NAME
+  user: POSTGRESQL_USERNAME
+  password: POSTGRESQL_PASSWORD
+```
+
+Replace POSTGRESQL_DATABASE_NAME, POSTGRESQL_USERNAME, and POSTGRESQL_PASSWORD with the name of your database, a user with read access to that database, and its credentials.
```
Comment (CaroFG) [inline learn/primary_db_sync/meilisync_postgresql.mdx:140]:
```suggestion
Replace `POSTGRESQL_DATABASE_NAME`, `POSTGRESQL_USERNAME`, and `POSTGRESQL_PASSWORD` with the name of your database, a user with read access to that database, and its credentials.
```

File: learn/primary_db_sync/meilisync_postgresql.mdx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+## Configure `meilisync`
+
+The last configuration step is specifying which tables in your database should be synced to which Meiliearch indexes.
```
Comment (CaroFG) [inline learn/primary_db_sync/meilisync_postgresql.mdx:144]:
```suggestion
The last configuration step is specifying which tables in your database should be synced to which Meilisearch indexes.
```

**Review by CaroFG — DISMISSED** [review]

File: learn/primary_db_sync/meilisync_postgresql.mdx [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+This will require a server configuration reload to take effect. Read more about this in the [`ALTER SYSTEM` documentation](https://www.postgresql.org/docs/current/sql-altersystem.html).
+
+### Connect `meilisync` and PostgreSQL
+
+Open `meilisync`'s `config.yml` once again and append the following settings to the bottom of the file:
+
+```yaml
+source:
+  type: postgres
+  host: 127.0.0.1
```
Comment (alallema) [inline learn/primary_db_sync/meilisync_postgresql.mdx:133]: I realize that I put the basic configuration if PostgreSQL is hosted on the same server as meilisync maybe we could add a note that the URL/host and port can be changed depending on the address of the Postgresql database. Not sure if it's mandatory

**Review by alallema — APPROVED** [review]
I just put a comment, but it can be published as is. Great work 🎉 
LGTM!

File: learn/primary_db_sync/meilisync_postgresql.mdx (hunk 2 above)
Comment (Strift) [reply learn/primary_db_sync/meilisync_postgresql.mdx:140]: I agree with this suggestion, but gui didn't do it in the other article so I'd rather keep it consistent.

File: learn/primary_db_sync/meilisync_postgresql.mdx (hunk 4 above)
Comment (Strift) [reply learn/primary_db_sync/meilisync_postgresql.mdx:133]: I added a note about this! Great catch, thanks

**Comment by Strift** [comment]
Thanks for your reviews @CaroFG @alallema 🙏 

@curquiza I think we're good to merge this. I don't know if you'd like to review it before :)

**Comment by curquiza** [comment]
@Strift I will not bring that much value by reviewing it 😅 

gui told me the following guidelines to merge your PR:
If you can wait for a week, please wait for gui's come back
If not, for example, if you have a deadline or anything, I will merge it!

File: learn/primary_db_sync/meilisync_postgresql.mdx [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+    index: MEILISEARCH_INDEX_NAME_2
+    full: true
+```
+
+Replace `POSTGRES_TABLE_NAME_1` and `MEILISEARCH_INDEX_NAME_1` with the names of your tables and indexes. As the above example demonstrates, you may sync multiple tables.
+
+<Capsule intent="tip" title="Customizing `meilisync`">
+  This tutorial describes a fairly basic `meilisync` setup. For more
+  customization options, consult [`meilisync`'s
+  documentation](https://github.com/long2ice/meilisync#configuration).
```
Comment (guimachiavelli) [inline learn/primary_db_sync/meilisync_postgresql.mdx:163]: Same as my comment on this bit on the other file.

**Review by guimachiavelli — CHANGES_REQUESTED** [review]

File: learn/primary_db_sync/meilisync_mysql.mdx [hunk 6]
```diff
@@ -146,7 +148,9 @@ sync:
 Replace `MYSQL_TABLE_NAME_1` and `MEILISEARCH_INDEX_NAME_1` with the names of your tables and indexes. As the above example demonstrates, you may sync multiple tables.
 
 <Capsule intent="tip" title="Customizing `meilisync`">
-This tutorial describes a fairly basic `meilisync` setup. For more customization options, consult [`meilisync`'s documentation](https://github.com/long2ice/meilisync#configuration).
+  This tutorial describes a fairly basic `meilisync` setup. For more
+  customization options, consult [`meilisync`'s
+  documentation](https://github.com/long2ice/meilisync#configuration).
```
[the remark this answers is not in the document]
Comment (Strift) [reply learn/primary_db_sync/meilisync_mysql.mdx:153]: This was not intentional, sorry 😬

**Comment by Strift** [comment]
Thanks for your review @guimachiavelli, I think I've addressed all your feedback 🙏

**Review by guimachiavelli — APPROVED** [review]
Thanks! LGTM!

**Pull request merged**